src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine+TransferInput.kt

```diff
@@ -25,6 +25,7 @@ enum class TransferInputField(val rawValue: String) {
     chain("chain"),
     token("token"),
     address("address"),
+    MEMO("memo"),
```

Comment:
lol this is for linting purposes, but yeah the other enum vals should be renamed at some point (would rather not incorporate into this PR)